src/modules/identity/RadixSimpleIdentity.spec.ts

```diff
@@ -38,7 +38,7 @@ describe('RadixSimpleIdentity', () => {
         expect(signature2).to.deep.equal(signature1)
     })
 
-    it('Signature of seeded identity can be verified OK', async () => {
+    it('signature of seeded identity can be verified OK', async () => {
```

Comment:
Ah! I wrote these tests! I didn't know the description was supposed to be all-low-caps
Thanks for cleaning up my mess 😅, I owe you a beer 🍺.